bin/enable-executable

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env ruby
+
+require 'fileutils'
+require 'pathname'
+
 puts 'Ensuring test files are executable...'
 
-Dir.glob('**/*test.rb').each do |f|
+test_files = Dir.glob(File.join(__dir__, '../exercises/**/*test.rb'))
+bin_files = Dir.glob(File.join(__dir__, '*'))
+
+(test_files + bin_files).each do |f|
   if File.executable?(f)
     print '.'
   else
-    print "Adding exec bit to #{f}."
-    File.chmod(0744, f)
+    relative_path = Pathname.new(f).relative_path_from(Pathname.pwd)
+    print "\nAdding exec bit to #{relative_path}."
```

Comment:
As far as I can tell we aren't adding exec bit to the relative path... only to a file.  the paths should already have exec bit otherwise we would not be able to change directory into them to get to the files.  This line may be saying the wrong thing because of the variable name.